internal/math32/sqrt.go

```diff
@@ -17,7 +17,9 @@ import (
 //	Sqrt(±0) = ±0
 //	Sqrt(x < 0) = NaN
 //	Sqrt(NaN) = NaN
-func Sqrt(x float32) float32 {
```

Comment:
This is the version that is used when asm isn't available.  
Use build tags to avoid duplicate definitions.  `!amd64 !arm64 noasm appengine safe`